backend/src/test/java/com/htecgroup/skynest/utils/UserObjectAccessKeyUtil.java

```diff
@@ -0,0 +1,12 @@
+package com.htecgroup.skynest.utils;
+
+import com.htecgroup.skynest.model.entity.UserObjectAccessKey;
+
+import java.util.UUID;
+
+public class UserObjectAccessKeyUtil {
+
+  public static final UserObjectAccessKey get() {
+    return new UserObjectAccessKey(UUID.randomUUID(), BucketEntityUtil.getPrivateBucket().getId());
```

Comment:
Again I think it is better to have some exact UUID instead of random, maybe if not now, then for later use :)